addons/binding/org.openhab.binding.netatmo/build.properties

```diff
@@ -11,7 +11,7 @@ bin.includes = META-INF/,\
                lib/okio-1.3.0.jar,\
                lib/json-20131018.jar,\
                lib/commons-codec-1.8.jar,\
-               lib/gson-2.3.1.jar,\
                about.html,\
-               lib/netatmo-java-retrofit-1.0.8.jar
+               lib/netatmo-java-retrofit-1.0.9.jar,\
```

Comment:
Git suggests (for the binary) that its renamed, which suggests that its same file, can you check it out?